### PR TITLE
feat(v2): add field deletion, duplication, set inactive functionality

### PR DESCRIPTION
**Pull request by mantariksh**
## Problem

Field deletion and duplication buttons currently don't do anything, and the `BiCog` icon to set the field to inactive is missing.

## Solution
- Add `BiCog` icon which sets field to inactive
- Add duplication and deletion functionality

### Set to inactive
field_inactive

### Duplicate
field_dup

### Delete
field_delete

### CODE REVIEW

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx [hunk 1]
```diff
@@ -176,14 +214,29 @@ export const FieldRowContainer = ({
...
                   spacing={0}
                 >
+                  <IconButton
+                    aria-label="Close field settings"
+                    icon={<BiCog fontSize="1.25rem" />}
+                    onClick={setToInactive}
+                  />
                   <IconButton
                     aria-label="Duplicate field"
+                    // Fields which are not yet created cannot be duplicated
```
Comment (karrui) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx:224]: Do we want to hide the duplicate button instead if state is creating field?

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx [hunk 2]
```diff
@@ -77,6 +98,23 @@ export const FieldRowContainer = ({
     [handleFieldClick],
   )
 
+  const handleDuplicateClick = useCallback(() => {
+    duplicateFieldMutation.mutate(field._id)
```
Comment (karrui) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx:102]: should we add a guard here to only allow duplicate if the field is in the creation stage?

**Review by karrui — APPROVED** [review]
lgtm

**Comment by karrui** [comment]
Oh yeah, `BiCog` to set inactive state seems weird...

**Comment by mantariksh** [comment]
@karrui yeah I already clarified this with pearly, afaik this is the intention of the design

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx [hunk 3]
```diff
@@ -176,14 +214,29 @@ export const FieldRowContainer = ({
                   colorScheme="secondary"
                   spacing={0}
                 >
+                  <IconButton
+                    aria-label="Close field settings"
+                    icon={<BiCog fontSize="1.25rem" />}
+                    onClick={setToInactive}
```
Comment (karrui) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx:220]: ohya this should be disabled state when queries are loading too

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx (hunk 1 above)
Comment (mantariksh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx:223]: ede3bdd8

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx (hunk 2 above)
Comment (mantariksh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx]: 7f67559a

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx (hunk 3 above)
Comment (mantariksh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx:224]: 09c6c655

**Pull request merged**